examples/agent_from_any_llm.py

```diff
@@ -43,10 +50,10 @@ def get_weather(location: str, celsius: Optional[bool] = False) -> str:
     return "The weather is UNGODLY with torrential rains and temperatures below -10°C"
 
 
-agent = ToolCallingAgent(tools=[get_weather], model=model)
+agent = ToolCallingAgent(tools=[get_weather], model=model, verbosity_level=2)
```

Comment:
```suggestion
agent = ToolCallingAgent(tools=[get_weather], model=model, verbosity_level=LogLevel.DEBUG)
```
Something that I have been fixing in my other PR. Since we use type hints.
The LogLevel comes from `monitoring.py`